capabilities/version-control.md

```diff
@@ -1,28 +1,61 @@
 # [Version Control](https://dora.dev/devops-capabilities/technical/version-control/)
 
-Version control allows teams to revert changes back to a working state. When this is fast an easy to accomplish, teams can move forward with less fear of breaking the system. Technologies like Git and SVN have made version control for code popular. However, this DORA Capability applies all the same to configuration, data, infrastructure, and documentation. Furthermore, version control is foundational for enabling automation, ensuring disaster recovery, and facilitating continuous integration.
+Version control allows teams to revert changes back to a working state. When this is fast and optimized for small, comprehensible changes, teams can move forward with the confidence that the system is up-to-date and stable. 
+
```

Comment:
What about when version control is limited or missing altogether? Suggest adding a line here that briefly describes what that looks like.